frontend/package.json

```diff
@@ -25,7 +25,7 @@
     "web-vitals": "^1.1.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "BROWSER='google-chrome-stable' react-scripts start",
```

Comment:
why did u do this, zostawmy bez to wtedy otwiera domyślną przeglądarkę

Reply:
pardon, u mnie jest jakaś lipa z domyślną przeglądarką i muszę to wklepywać